salsah/src/public/js/jquery.searchbox.js

```diff
@@ -217,6 +224,15 @@
 								$this.css({cursor: 'text'});
 							}
 							localdata.ele.sel.hide();
+							// when leaving this input, if the value is empty, but we had a previous valid input
+							// set back the previous input
+							if ($this.attr("isValid") && $this.attr("isValid") == "false") {
```

Comment:
In general, I prefer to write `$this.attr("isValid") !== undefined` (in case the requested attribute isn't set, `undefined` would be returned).